implementations/rust/ockam/ockam_command/src/status/mod.rs

```diff
@@ -22,8 +22,15 @@ use ockam_api::nodes::{BackgroundNodeClient, InMemoryNode};
 use ockam_api::output::Output;
 use ockam_api::{fmt_heading, fmt_log, fmt_separator, fmt_warn};
 
+const LONG_ABOUT: &str = include_str!("./static/long_about.txt");
+const AFTER_LONG_HELP: &str = include_str!("./static/after_long_help.txt");
+
 /// Display information about the system's status
 #[derive(Clone, Debug, Args)]
+#[command(
```

Comment:
Perhaps it would be better to replace "Display information about the system's status" by "Display information about your Ockam instance"